src/front-end/src/components/MentorForm.js

```diff
@@ -63,24 +63,33 @@ class MentorForm extends Component {
               label="Name"
               name="nameInput"
               // must have an input:
-              rules={[{ required: true, message: 'Please input something' }]}
+              rules={[{ required: true, message: 'Please enter your name' }]}
             >
-              <Input placeholder="please enter your name" />
+              <Input placeholder="First name Last name" />
             </Form.Item>
             <Form.Item
               label="Email"
               name="emailInput"
-              type="email"
               // must have an input:
-              rules={[{ required: true, message: 'Please input something' }]}
+              rules={[
+                {
+                  required: true,
+                  message: 'Please enter your email in a proper format',
+                },
+              ]}
             >
-              <Input placeholder="please enter your email" />
+              <Input placeholder="name@example.com" />
```

Comment:
Please also add 'type="email"' to the Input tag

Reply:
I just noticed that for some reason github wasn't displaying the Input tag in my comment from earlier (I had to add a '#' at the start of the tag). Sorry, I didn't catch that before! If you copy paste all of this (minus the # in the tag) it should work
<Form.Item
label="Email"
name="emailInput"
// must have an input:
rules={[{ required: true, message: 'Please input something' }]}
>
<#Input placeholder="please enter your email" type="email" />
</Form.Item>

Reply:
done